Library/Homebrew/cmd/bottle.rb

```diff
@@ -192,10 +204,24 @@ def bottle_formula(f)
 
         keg.delete_pyc_files!
 
+        tab = Tab.for_keg(keg)
+        empty_tab = Tab.empty
+        tab["poured_from_bottle"] = empty_tab["poured_from_bottle"]
+        tab["HEAD"] = empty_tab["HEAD"]
+        tab["time"] = empty_tab["time"]
+        tab.write
+
+        keg.find {|k| File.utime(File.atime(k), formula_source_time, k) }
```

Comment:
Yes. It doesn't seem to be easy to do (at least directly from Ruby) and it might turn out it's more practical to manipulate the created `tar` archive.